tests/pytests/unit/beacons/test_salt_proxy.py

```diff
@@ -0,0 +1,76 @@
+# Python libs
+
+from collections import namedtuple
+
+import pytest
+
+# Salt libs
+import salt.beacons.salt_proxy as salt_proxy
+from tests.support.mock import MagicMock, patch
+
+PATCH_OPTS = dict(autospec=True, spec_set=True)
```

Comment:
I'm not seeing `PATCH_OPTS` or `FakeProcess` being used anywhere. We also don't want to be using mutable objects at the global level.